README.md

```diff
@@ -124,12 +118,44 @@ For in-depth examples of NNCF integration, browse the [sample scripts](#model-co
 For FAQ, visit this [link](./docs/FAQ.md).
 
 
+## Usage of Post-Training Quantization
+
+The basic workflow steps:
+1) Initialize the dataset transformation function.
+2) Initialize NNCF Dataset.
+3) Run the quantization pipeline.
+
+NNCF provides [samples](#post-training-quantization) that demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
+
+```python
+import onnx
+import nncf
+import torch
+import torchvision.datasets as datasets
+
+# Instantiate your uncompressed model
+onnx_model = onnx.load_model('/model_path')
+# Provide validation part of the dataset for statistics collection for compression algorithm
+representative_dataset = datasets.ImageFolder("/path")
```

Comment:
What's wrong with the quotes? Two rows above you used singles.

Reply:
Also, how user's dataset can be not representative?

Reply:
I suppose it's because the dataset here is not necessarily the one used for training, or not necessarily the complete dataset.

Reply:
changed to `calibration_dataset`

Reply:
It is not changed here.